libs/logger/formatters/FormFormatter.php

```diff
@@ -2,26 +2,30 @@
 
 namespace Leaf;
 
-class FormFormatter
-{
+class FormFormatter{

```

Comment:
Are you moving the { up to the same line as the class definition or is your IDE? I've noticed it in some instances but not all.

 PSR12 - 4.1 says

The opening brace for the class MUST go on its own line; the closing brace for the class MUST go on the next line after the body.

Opening braces MUST be on their own line and MUST NOT be preceded or followed by a blank line.

Reply:
Thought I fixed this already. Will fix again